### DISCUSSION TITLE
confusion: what does Pod Networking stand for in the docs ?

### DISCUSSION THREAD

**Discussion by luckydogxf** [ceph/ceph-csi#4393] [Q&A] [answered]
The current problem with Pod Networking, is when a CephFS/RBD/NFS volume is mounted in a pod using Ceph CSI and then the CSI CephFS/RBD/NFS plugin is restarted or terminated (e.g. by restarting or deleting its DaemonSet), all operations on the volume become blocked, even after restarting the CSI pods.

Hi， Here the term Pod Network equals to 'Host Network'[1] ?
I used calico as po network plugin. I didn't find volumes was stuck after I removed po named csi-rbdplugin-XXX.
Judge from the descriptions, I guess this may be related to Hostnetwork.

[1] https://www.alibabacloud.com/help/en/ack/ack-managed-and-ack-dedicated/user-guide/use-the-host-network

can you give a detailed explanation? thanks

**Comment by Madhu-1** [comment, marked as answer]
@luckydogxf here PodNetworking referring to the actual network used for kubernetes pods like CNI (calico), but the templates are not updated for pod networking, the current templates still use the host networking, not pod networking. Currently Rook is the only users of this feature.  we even might drop the support for it in next few releases due to different problem, more details here https://github.com/rook/rook/issues/13055
